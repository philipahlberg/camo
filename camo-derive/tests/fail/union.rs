use camo_derive::Camo;

#[derive(Camo)]
union Foo {
    foo: u32,
    bar: bool,
}

fn main() {}
