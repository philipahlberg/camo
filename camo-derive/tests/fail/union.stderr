error: `camo` does not support unions
 --> tests/fail/union.rs:4:1
  |
4 | union Foo {
  | ^^^^^
